tests/test_make_game.py

```diff
@@ -11,11 +11,11 @@ def test_making_game_with_names_to_exclude():
     g_rng.set_seed(42)
 
     with make_temp_directory(prefix="test_render_wrapper") as tmpdir:
-        game_file1, game1 = textworld.make(2, 20, 3, {"names_to_exclude": []},
+        game_file1, game1 = textworld.make(2, 20, 3, 3, {"names_to_exclude": []},
```

Comment:
That's a lot of numbers in a row with no obvious meaning.  Maybe we should make some options class to hold them?  It could include the grammar flags and everything too.

Reply:
Yes, I think it makes a lot of sense to do that. I'll follow the way you've done it with the chaining `Options`.

Reply:
This is implemented in #44